resources/views/MyAccount/Components/OrderDocuments.twig

```diff
@@ -0,0 +1,31 @@
+{% set enabledRoutes = config("IO.routing.enabled_routes") | split(', ') %}
+
+<script type="x/template" id="vue-order-documents">
+    <div>
+        {% if "order-document" in enabledRoutes or "all" in enabledRoutes %}
+            <template v-if="type == 'order'">
+                <div v-for="receipt of activeDocuments">
+                    <a class="btn btn-primary btn-block" :href="'/order-document/' + receipt.id" target="_blank">
+                        ${ getTypeName(receipt.type) }
+                        <i class="fa fa-file-text" aria-hidden="true"></i>
+                    </a>
+                </div>
+            </template>
+
+            <template v-else>
+                <template v-for="document of activeDocuments">
+                    <div v-if="document.type == 'return_note'">
+                        <div class="icons d-none d-sm-block">
+                            <a target="_blank" :class="'text-' + appearance" :href="'/order-document/' + document.id" @click.stop v-tooltip data-toggle="tooltip" data-placement="top" title="{{ trans('Ceres::Template.returnHistory
+                            ') }}"><i class="fa fa-file-text-o" aria-hidden="true"></i></a>
+                        </div>
+                        <div class="buttons d-block d-sm-none mt-2">
+                            <a target="_blank" :class="'btn-' + appearance" :href="'/order-document/' + document.id" @click.stop class="btn btn-block"><span>{{ trans("Ceres::Template.returnHistoryReceipt") }}</span><i class="fa fa-file-text-o" aria-hidden="true"></i></a>
+                        </div>
+                    </div>
+                    <p v-else="document.type == 'return_note'">RababerRababer</p>
+                </template>
+            </template>
+        {% endif %}
+    </div>
+</script>
```

Comment:
Warum wird auf Template-Seite überhaupt zwischen orderType unterschieden? Kann nicht ein generisches Template für beide Fälle gebaut werden?

Reply:
Das Anzeigen der Dokumente ist in beiden Fällen unterschiedlich. Mal Buttons mal Icons

Reply:
Redundantes Markup entfernt